### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by YueDai0514** [hxwork/OMNet_Pytorch#12]
Hi
I'm sorry to bother you. When I set the transform_type to rpmnet_noise, I found that the test results exceeded the metrics you posted in the Readme file when I trained for 3000 epochs, may I ask if you must have trained for 10,000 epochs to get the results, and also may I ask if the data in the readme is the best result?

**Comment by hxwork** [comment]
Hi,

Glad to hear that you can obtain better results using our method! Actually, the performance in README is the best result I obtained before the publication of this paper. You can stop the training if you have obtained better results before 10,000 epochs. OR, you can also let it train for a longer time to see if better results can be produced. The number of training epochs is a hyperparameter that you can adjust.

**Comment by jfjjdsjdj** [comment]
Hi，
Thank you very much for your reply. I just started to study point cloud registration. I still have some questions about OMNet. First, I want to ask you about the prediction module of overlap mask. Why can the features after the hybrid feature extraction module be predicted by the multi-layer perceptron (the output layer channel is 2)? In more detail, my question is the global features of the source point cloud and the features between the local features and the global features of the target point cloud extracted by the hybrid module. What is the principle of classification prediction through the perceptron? Is it because the features of overlapping parts are more obvious after mixed feature extraction, so they are classified with obvious and inobvious differences? The second question is the same, why can we directly regress quaternion and translation vector by the characteristics of source point cloud and target point cloud after information fusion? I don't understand the principle. Please forgive me if my question is too stupid. Looking forward to your reply.
